Assets/_Game/Scripts/Fungus/ItemExists.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using Fungus;
+
+[CommandInfo("DLS",
+             "Item Exists",
+             "Check an Inventory if an item exists.")]
+[AddComponentMenu("")]
+public class ItemExists : Command
+{
+    [SerializeField] protected InventoryObject inventorySource;
+    [SerializeField] protected string itemName;
+    [VariableProperty(typeof(BooleanVariable))]
+    [SerializeField] protected Variable exists;
+    public override void OnEnter()
+    {
+        if(inventorySource != null)
+        {
+            exists.Apply(SetOperator.Assign, inventorySource.ItemExists(itemName));

```

Comment:
Suggestion: It would probably be good in the future to look into emulating the if/else command rather than setting a variable and having a straight if to check it. Remove complexity for whoever is going to be inputting the stuff in the editor